### DISCUSSION TITLE
contributing/workflow/pr_workflow

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#133] [User-contributed notes]
# contributing/workflow/pr_workflow

The so-called "PR workflow" used by Godot is common to many projects using Git, and should be familiar to veteran free software contributors. The idea is that only a small number (if any) commit di...

https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html

**Comment by shahriarlabib000** [comment]
I think `$` should be removed makes it hard to copy/paste

**Reply by Calinou** [reply]
I agree, I've opened a PR accordingly:

- https://github.com/godotengine/godot-docs/pull/9956

**Reply by shahriarlabib000** [reply]
Thank you
